package/README.md

```diff
@@ -1,9 +1,13 @@
 <p align="center">
-  <a href="https://sentry.io" target="_blank" align="center">
-    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  <a href="https://sentry.io" target="_blank" align="left">
+    <img src="https://raw.githubusercontent.com/getsentry/sentry-unity/main/.github/sentry-wordmark-dark-400x119.svg" width="280">
+  </a>
+  <a href="https://docs.sentry.io/platforms/unity/" target="_blank" align="right">
+    <img src="https://raw.githubusercontent.com/getsentry/sentry-unity/main/.github/unity-verified-logo.svg" width="280">
```

Comment:
maybe use arelative image path?
```suggestion
    <img src="./.github/sentry-wordmark-dark-400x119.svg" width="280">
  </a>
  <a href="https://docs.sentry.io/platforms/unity/" target="_blank" align="right">
    <img src="./.github/unity-verified-logo.svg" width="280">
```

Reply:
This readme ends up on the package-repo https://github.com/getsentry/unity and relative paths won't work there.